### PR TITLE
Bugs/ChatGroup/CLK32-fix popup + avatar detail not show

### PR DESCRIPTION
**Pull request by Minhdn1**
## Prerequisite
- [x] Self-review :monocle_face:
- [x] No Crashes ☠️
---

### Root cause
- // Explain what you have done

### Solution
- // Explain how you fix it

### Impact
- // List of affected components/modules

### Screenshots (Optional)
- // Capture affected screens

### CODE REVIEW

File: ClearKeep/App/Resources/Localize.strings [hunk 1]
```diff
@@ -270,6 +271,9 @@
 "GroupDetail.TitleCheckbox" = "Add user from other server";
 "GroupDetail.UserInitTitle" = "Users int his Group chat";
 "GroupDetail.Title.PlaceHoder" = "Paste your friend's link";
+"GroupDetail.Remove" = "Remove";
+"GroupDetail.LeadMesage.Remove" = " Do you want to remove ";
+"GroupDetail.Message.Remove" = "from the group chat?";
```
Comment (vmo-klinh) [inline ClearKeep/App/Resources/Localize.strings]: Use localize with parameter

File: ClearKeep/Scenes/GroupDetail/Views/RemoveMemberView.swift [hunk 2]
```diff
@@ -116,7 +129,11 @@ private extension RemoveMemberView {
 	}
 
 	func search(text: String) {
-		self.clientData = clientData.filter { $0.userName.lowercased().prefix(1) == text.lowercased().prefix(1) }
+		self.member = member.filter { $0.displayName.lowercased().prefix(1) == text.lowercased().prefix(1) }
+	}
+
+	var messagePopup: String {
+		"GroupDetail.LeadMesage.Remove".localized + "\(groupData.first?.userName ?? "") " + "GroupDetail.Message.Remove".localized
```
Comment (vmo-klinh) [inline ClearKeep/Scenes/GroupDetail/Views/RemoveMemberView.swift]: Use localize string with format

**Review by vmo-klinh — CHANGES_REQUESTED** [review]

File: ClearKeep/Scenes/GroupDetail/Views/RemoveMemberView.swift (hunk 2 above)
Comment (Minhdn1) [reply ClearKeep/Scenes/GroupDetail/Views/RemoveMemberView.swift]: fix complete

File: ClearKeep/App/Resources/Localize.strings (hunk 1 above)
Comment (Minhdn1) [reply ClearKeep/App/Resources/Localize.strings]: fix complete

**Review by vmo-klinh — APPROVED** [review]

**Pull request merged**